src/pybind/mgr/dashboard/frontend/src/app/core/navigation/notifications/notifications.component.scss

```
@@ -7,3 +7,20 @@
 .running:hover i {
   color: bd.$color-solid-white;
 }
+
+a {
+  .dot {
+    background-color: bd.$color-blue;
```

We should use the same color we use when there are running tasks:
```suggestion
    background-color: bd.$color-primary;
```